.github/ISSUE_TEMPLATE/story.yaml

```
@@ -0,0 +1,43 @@
+# Copyright 2022 Z5Labs and Contributors
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Story
+description: Represents stories
+title: "story(subject): short description"
+labels: ["story"]
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Please provide an informal, general explanation of the work needed to be done, written from the perspective of the end user.
+    validations:
+      required: true
+
+  - type: textarea
+    id: acceptance-criteria
+    attributes:
+      label: Acceptance Criteria
+      description: List the criteria for accepting this story as done
+    validations:
+      required: true
+
+  - type: textarea
```

```suggestion
  - type: markdown
```
GH flavored markdown allows you to reference issues by #. Text areas are not suitable for this input type.